Users/Tutorial.rst

```diff
@@ -351,7 +351,7 @@ To get help on using a bear or to get a description of the bear, use the
     coala --bears=SpaceConsistencyBear --show-bears --show-description
```

Comment:
I was trying to open the issue again to get the full url, but it opens this page only.
How can I get the full url now?